porick/lib.py

```diff
@@ -1,12 +1,48 @@
 import bcrypt
 import re
+import smtplib
+
+from email.mime.text import MIMEText
 
 from flask import request
 from sqlalchemy import or_
 
 from models import User
 from porick import db
-from settings import PASSWORD_SALT
+from settings import PASSWORD_SALT, SERVER_DOMAIN, PASSWORD_RESET_REQUEST_EXPIRY, SMTP_REPLYTO, SMTP_SERVER
+
+
+
+reset_password_text = """
+Hi,
+
+A password reset has been requested for your account on Porick.
+
+To reset your password, please click the link below.
+
+http://{server_domain}/reset_password?key={key}
+
+This URL will be valid for {validity}.
+
+If you did not initiate this password reset then you may simply disregard this email.
+
+Cheers,
+Porick
+
+"""
+
+def send_reset_password_email(user_email, key):
+    validity = '{} hour{}'.format(PASSWORD_RESET_REQUEST_EXPIRY, '' if PASSWORD_RESET_REQUEST_EXPIRY == 1 else 's')
```

Comment:
But is incorrect if PASSWORD_RESET_REQUEST_EXPIRY is 0! :O . But yeah, I'll go with that.